shell/app-shell/elements/remote-shared-handles.js

```diff
@@ -17,28 +17,34 @@ class RemoteSharedHandles extends Xen.Base {
   static get observedAttributes() { return ['arc', 'friends', 'user']; }
   _getInitialState() {
     return {
-      group: Object.assign(new WatchGroup(), {db})
+      group: new WatchGroup()
     };
   }
   _update(props, state, lastProps) {
     // TODO(sjmiles): rely on invariant that `arc` and `user` are required a-priori for `friends`?
     if (/*props.arc && props.user &&*/ props.friends !== lastProps.friends) {
       let watches = [];
       if (props.friends && props.user) {
-        let friends = props.friends.map(friend => friend.rawData);
+        // TODO(sjmiles): making a null-prototype object makes it easeir to read in console (roi?)
+        //let friends = props.friends.map(friend => friend.rawData);
```

Comment:
Do we need this commented out bit?

Reply:
I wanted to leave it for a iteration or two, in case I decide the ROI is there or not. I tend to do this from time to time, I can restrain myself if we consider it untidy.